docs/user/guides/generate-bindings.md

```diff
@@ -1,75 +1,75 @@
+1. From a running pulp instance:
 
-### Remote Systems
+    ```bash
+    PULP_URL="http://localhost:24817/pulp/api/v3/"
+    COMPONENT="core"
+    URL="${PULP_URL}docs/api.json?bindings&component=${COMPONENT}"
+    curl "${URL}" -o "${COMPONENT}-api.json"
+    ```
 
-During bindings generation the openapi schema is fetched. Use the `PULP_API` environment
-variable to instruct the bindings generator to use a Pulp API on a different host and/or port.
-For example, `export PULP_API="http://localhost:24817"` are the default host and port, which
-results in the bindings generator talking to the Pulp API at
-`http://localhost:24817/pulp/api/v3/docs/api.json`.
+## 3) Generate the client
 
-### Local Openapi Schema
+The script uses the [openapi-generator-cli image] to generate the clients, so the first run may take some time to download it.
 
-If you want to use a locally present openapi schema, you can skip fetching the openapi schema
-by setting the `USE_LOCAL_API_JSON` environment variable. Doing so you have to manually provide the
-`api.json` file containing the openapi schema in the current working directory.
+In the example, a ruby package for pulpcore will be generated at `./pulp_rpm-client/` using an existing schema `rpm-api.json`:
 
-### Docker in Docker (dind)
+```bash
+./gen-client.sh rpm-api.json rpm ruby
+```
 
-Bindings are generated using the openapi-generator-cli docker container. If your environment itself runs in
-a docker container, the openapi-generator-cli container has to be started as a sibling container. For
-sibling containers, volumes cannot be mounted as usual. They have to be passed through from the parent
-container. For this to work you have to set the `PARENT_CONRAINER_ID` environment variable to specify the
-parent container in a dind environment.
+## Further reading
 
-### Filesystem Shared With Another Container
+- For Docker-in-Docker (dind) environments, see the [PARENT_CONTAINER_ID] setting.
+- For filesystem shared with another container, see the [PULP_MCS_LABEL] setting.
+- If you are upgrading pulpcore to `>3.70`, check the [migration guide].
 
-When the bindings are being generated so that they can be installed inside another container, it
-may be necessary to set the MCS label on the openapi-generator-cli container to match the MCS label
-of the other container. Users can set the \$PULP_MCS_LABEL environment variable (e.g. s0:c1,c2).
-When this variable is present, the container for `openapi-generator-cli` will be started with this
-MCS label. This only applies to systems that are using `podman` and SELinux is `Enforcing`.
+[how it works]: site:pulp-openapi-generator/docs/user/learn/how-it-works/
+[migration guide]: site:pulp-openapi-generator/docs/user/guides/version-migrations/
+[openapi-generator-cli image]: https://openapi-generator.tech/docs/installation/#docker
+[parent_container_id]: site:pulp-openapi-generator/docs/user/reference/settings/#parent_container_id
+[pulp-glue]: site:pulp-glue/docs/dev/
```

Comment:
As much as i like the fact that you want to reference this here, I don't see it in the text...

Reply:
Found it this time. XD